doc/modules/ensemble.rst

```diff
@@ -440,61 +440,39 @@ Gradient Tree Boosting
 ======================
 
 `Gradient Tree Boosting <https://en.wikipedia.org/wiki/Gradient_boosting>`_
-or Gradient Boosted Regression Trees (GBRT) is a generalization
+or Gradient Boosted Degression Trees (GBDT) is a generalization
```

Comment:
For a second there I was wondering if "Degression Trees" were a thing that I should read about :)

Reply:
Oops ^^ thanks